src/components/Layouts/_index.scss

```diff
@@ -0,0 +1,104 @@
+////
+/// Modular layouts styles.
```

Comment:
For any/all mixins, classes, and functions in this file, I'm very open to renaming 😆